### ISSUE TITLE
NFT the internet RFCs and start paying attribution to them

### ISSUE THREAD

**Issue by inverted-capital** [dreamcatcher-tech/dreamcatcher-tech.github.io#29]
In [this list of RFCs](https://en.wikipedia.org/wiki/List_of_RFCs) the foundations of the internet can be seen to be built up over time.  RFCs are Requests in our model, and they also have an author attached to them.  

At great human and economic cost, resources drawn from the U.S. Government, industry and the academic community were assembled into a collection of interconnected networks called the Internet.  We should NFT all these RFCs and we should begin to pay attribution to this machine we depend upon so strongly.

**Comment by TemperBead** [comment]
Agree.  Where do we put this proposal?  We shouldn't keep it in issues but needs to go somewhere.

**Comment by inverted-capital** [comment]
What about as a Request the we can use as the basis for all Requests that require Internet as a Dependency ?  This would be the perfect use case for Inheritance as requested in #36 

I assume that an Idea that can satisfy this Request would have to wait until we have an app ?  But we can certainly propose ideas in markdown, nonetheless.  To be useful, this Request should limit its scope somehow, to say pre 1981 or similar, because this time period seems easiest to trace.

**Comment by inverted-capital** [comment]
Sorry I didn't actually mean to close this, but I typed the commit message wrong
